test/run_test.sh

```diff
@@ -203,6 +208,9 @@ function link_latest() {
 # If an outcome file was produced, keep a link to the latest file
 link_latest outcome json
 
+# If a traffic model file was produced, keep a link to the latest file
+link_latest traffic.model json
```

Comment:
Yes, if the output file is empty, you want to know!
And we can exclude noisy lines (like the current time) if we need to.